extractor/src/main/java/org/schabi/newpipe/extractor/services/peertube/extractors/PeertubeChannelExtractor.java

```diff
@@ -59,12 +59,12 @@ public String getFeedUrl() throws ParsingException {
 
     @Override
     public long getSubscriberCount() throws ParsingException {
-        Number number = JsonUtils.getNumber(json, "followersCount");
+        final Number number = JsonUtils.getNumber(json, "followersCount");
         return number.longValue();
```

Comment:
No no no.

```suggestion
        final Number number = JsonUtils.getNumber(json, "followersCount");
        return number == null ? -1 /* or throw a clear exception of what's going on??? */ : number.longValue();
```
In Invidious PR #352, I got some NullPointerException s because getNumber() returns null by default, unlike getObject() and getArray().

Reply:
That's why it should be `json.getLong("followersCount")`

Reply:
It is null safe but return 0 instead of -1, you need to take on board this.

Reply:
@B0pol: Yes. This code is basically duplicate in a lot of places. In some places, the old behavior was to fall back to the previously fetched page (or 0; that's just stupid) or throw a NullPointerException. Also, in SoundCloud, 0 was often explicitly defined as fallback.

Reply:
We should return -1, but that should be done in a separate PR.